pkg/config/eventstore.go

```diff
@@ -49,3 +53,17 @@ type EventStorePulseConfig struct {
 type EventStoreAxiomConfig struct {
 	Dataset ValueSource `yaml:"dataset"`
 }
+
+type EventStoreOTelConfig struct {
+	Endpoint    string            `yaml:"endpoint"`
+	Protocol    string            `yaml:"protocol"` // "grpc", "http", "stdout"
+	Headers     map[string]string `yaml:"headers"`  // Optional headers for auth
```

Comment:
```suggestion
	Headers     map[string]ValueSource `yaml:"headers"`  // Optional headers for auth
```

let's use this so we can source auth tokens externally